lua_cjson.c

```diff
@@ -75,6 +75,17 @@
 #define DEFAULT_DECODE_INVALID_NUMBERS 0
 #endif
 
+/*
+ * 
```

Comment:
Why this line? Also this line has a line trailing space which shouldn't be there.